packages/maskbook/src/plugins/Wallet/services/assets.ts

```diff
@@ -1,10 +1,24 @@
 import { AssetProvider } from '../types'
 import * as OpenSeaAPI from '../apis/opensea'
+import type { AssetInCard } from '../../../plugins/Wallet/apis/opensea'
 
 export async function getAssetsListNFT(address: string, provider: AssetProvider) {
     if (provider === AssetProvider.OPENSEAN) {
         const { assets } = await OpenSeaAPI.getAssetsList(address)
-        return assets
+        return assets.map(
+            (x) =>
```

Comment:
Use `(x): AssetInCard => ({})` instead of `(x) => ({} as AssetInCard)`. The prior one is stricter